apps/emqx_conf/test/emqx_conf_app_SUITE.erl

```diff
@@ -35,19 +35,27 @@ t_copy_conf_override_on_restarts(Config) ->
     ),
 
     %% 1. Start all nodes
+    ct:pal("starting cluster"),
     Nodes = start_cluster(Cluster),
     try
+        ct:pal("checking state"),
         assert_config_load_done(Nodes),
 
         %% 2. Stop each in order.
+        ct:pal("stopping cluster"),
         stop_cluster(Nodes),
 
         %% 3. Restart nodes in the same order.  This should not
         %% crash and eventually all nodes should be ready.
+        ct:pal("restarting cluster"),
         restart_cluster_async(Cluster),
 
-        timer:sleep(15000),
+        ct:pal("waiting for stabilization"),
+        PrevFlag = process_flag(trap_exit, true),
+        wait_emqx_conf_started(Nodes, 35_000),
+        process_flag(trap_exit, PrevFlag),
 
+        ct:pal("checking state"),
         assert_config_load_done(Nodes),
```

Comment:
I'm not sure this test case makes much sense as of today, since we [inhibit the conventional config loader](https://github.com/emqx/emqx/blob/361490005b50dc6aacc30f04fbbd40ffb00d0ac8/apps/emqx/test/emqx_cth_suite.erl#L366).  :thinking: